<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmMain</class>
 <widget class="QMainWindow" name="FrmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>432</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>881</width>
    <height>432</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>881</width>
    <height>432</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Little Teapot</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/images/icons/teapot.ico</normaloff>:/images/images/icons/teapot.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/*
	Copyright 2013 Emanuel Claesson

	Licensed under the Apache License, Version 2.0 (the &quot;License&quot;);
	you may not use this file except in compliance with the License.
	You may obtain a copy of the License at

		http://www.apache.org/licenses/LICENSE-2.0

	Unless required by applicable law or agreed to in writing, software
	distributed under the License is distributed on an &quot;AS IS&quot; BASIS,
	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
	See the License for the specific language governing permissions and
	limitations under the License.
*/

/*
	COLOR_DARK     = #191919
	COLOR_MEDIUM   = #353535
	COLOR_MEDLIGHT = #5A5A5A
	COLOR_LIGHT    = #DDDDDD
	COLOR_ACCENT   = #3C4E77
*/

* {
	background: #191919;
	color: #DDDDDD;
	border: 1px solid #5A5A5A;
}

QWidget::item:selected {
	background: #3C4E77;
}

QCheckBox, QRadioButton {
	border: none;
}

QRadioButton::indicator, QCheckBox::indicator {
	width: 13px;
	height: 13px;
}

QRadioButton::indicator::unchecked, QCheckBox::indicator::unchecked {
	border: 1px solid #5A5A5A;
	background: none;
}

QRadioButton::indicator:unchecked:hover, QCheckBox::indicator:unchecked:hover {
	border: 1px solid #DDDDDD;
}

QRadioButton::indicator::checked, QCheckBox::indicator::checked {
	border: 1px solid #5A5A5A;
	background: #5A5A5A;
}

QRadioButton::indicator:checked:hover, QCheckBox::indicator:checked:hover {
	border: 1px solid #DDDDDD;
	background: #DDDDDD;
}

QGroupBox {
	margin-top: 6px;
}

QGroupBox::title {
	top: -7px;
	left: 7px;
}

QScrollBar {
	border: 1px solid #5A5A5A;
	background: #191919;
}

QScrollBar:horizontal {
	height: 15px;
	margin: 0px 0px 0px 32px;
}

QScrollBar:vertical {
	width: 15px;
	margin: 32px 0px 0px 0px;
}

QScrollBar::handle {
	background: #353535;
	border: 1px solid #5A5A5A;
}

QScrollBar::handle:horizontal {
	border-width: 0px 1px 0px 1px;
}

QScrollBar::handle:vertical {
	border-width: 1px 0px 1px 0px;
}

QScrollBar::handle:horizontal {
	min-width: 20px;
}

QScrollBar::handle:vertical {
	min-height: 20px;
}

QScrollBar::add-line, QScrollBar::sub-line {
	background:#353535;
	border: 1px solid #5A5A5A;
	subcontrol-origin: margin;
}

QScrollBar::add-line {
	position: absolute;
}

QScrollBar::add-line:horizontal {
	width: 15px;
	subcontrol-position: left;
	left: 15px;
}

QScrollBar::add-line:vertical {
	height: 15px;
	subcontrol-position: top;
	top: 15px;
}

QScrollBar::sub-line:horizontal {
	width: 15px;
	subcontrol-position: top left;
}

QScrollBar::sub-line:vertical {
	height: 15px;
	subcontrol-position: top;
}

QScrollBar:left-arrow, QScrollBar::right-arrow, QScrollBar::up-arrow, QScrollBar::down-arrow {
	border: 1px solid #5A5A5A;
	width: 3px;
	height: 3px;
}

QScrollBar::add-page, QScrollBar::sub-page {
	background: none;
}

QAbstractButton:hover {
	background: #353535;
}

QAbstractButton:pressed {
	background: #5A5A5A;
}

QAbstractItemView {
	show-decoration-selected: 1;
	selection-background-color: #3C4E77;
	selection-color: #DDDDDD;
	alternate-background-color: #353535;
}

QHeaderView {
	border: 1px solid #5A5A5A;
}

QHeaderView::section {
	background: #191919;
	border: 1px solid #5A5A5A;
	padding: 4px;
}

QHeaderView::section:selected, QHeaderView::section::checked {
	background: #353535;
}

QTableView {
	gridline-color: #5A5A5A;
}

QTabBar {
	margin-left: 2px;
}

QTabBar::tab {
	border-radius: 0px;
	padding: 4px;
	margin: 4px;
}

QTabBar::tab:selected {
	background: #353535;
}

QComboBox::down-arrow {
	border: 1px solid #5A5A5A;
	background: #353535;
}

QComboBox::drop-down {
	border: 1px solid #5A5A5A;
	background: #353535;
}

QComboBox::down-arrow {
	width: 3px;
	height: 3px;
	border: 1px solid #5A5A5A;
}

QAbstractSpinBox {
	padding-right: 15px;
}

QAbstractSpinBox::up-button, QAbstractSpinBox::down-button {
	border: 1px solid #5A5A5A;
	background: #353535;
	subcontrol-origin: border;
}

QAbstractSpinBox::up-arrow, QAbstractSpinBox::down-arrow {
	width: 3px;
	height: 3px;
	border: 1px solid #5A5A5A;
}

QSlider {
	border: none;
}

QSlider::groove:horizontal {
	height: 5px;
	margin: 4px 0px 4px 0px;
}

QSlider::groove:vertical {
	width: 5px;
	margin: 0px 4px 0px 4px;
}

QSlider::handle {
	border: 1px solid #5A5A5A;
	background: #353535;
}

QSlider::handle:horizontal {
	width: 15px;
	margin: -4px 0px -4px 0px;
}

QSlider::handle:vertical {
	height: 15px;
	margin: 0px -4px 0px -4px;
}

QSlider::add-page:vertical, QSlider::sub-page:horizontal {
	background: #3C4E77;
}

QSlider::sub-page:vertical, QSlider::add-page:horizontal {
	background: #353535;
}

QLabel {
	border: none;
}

QProgressBar {
	text-align: center;
}

QProgressBar::chunk {
	width: 1px;
	background-color: #3C4E77;
}

QMenu::separator {
	background: #353535;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="Box" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>741</width>
        <height>389</height>
       </size>
      </property>
      <widget class="ModelViewer" name="modelViewer" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>371</width>
         <height>391</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="ModelViewer" name="modelViewer2" native="true">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>0</y>
         <width>371</width>
         <height>391</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>16777215</height>
       </size>
      </property>
      <widget class="QGroupBox" name="grpPosition">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>121</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Position</string>
       </property>
       <widget class="QDoubleSpinBox" name="dsbPositionX">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>87</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="lblPositionY">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lblPositionX">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>16</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="dsbPositionY">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>87</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
       <zorder>dsbPositionX</zorder>
       <zorder>lblPositionY</zorder>
       <zorder>lblPositionX</zorder>
       <zorder>dsbPositionY</zorder>
       <zorder>widget_2</zorder>
      </widget>
      <widget class="QGroupBox" name="grpRotation">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>121</width>
         <height>111</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Rotation</string>
       </property>
       <widget class="QDoubleSpinBox" name="dsbRotationX">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>87</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="suffix">
         <string>° </string>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="lblRotationY">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lblRotationX">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>16</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="dsbRotationY">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>87</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="suffix">
         <string>° </string>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="dsbRotationZ">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>87</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="suffix">
         <string>° </string>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="lblRotationZ">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Z:</string>
        </property>
       </widget>
       <zorder>dsbRotationX</zorder>
       <zorder>lblRotationY</zorder>
       <zorder>lblRotationX</zorder>
       <zorder>dsbRotationY</zorder>
       <zorder>dsbRotationZ</zorder>
       <zorder>lblRotationZ</zorder>
       <zorder>modelViewer</zorder>
      </widget>
      <widget class="QGroupBox" name="grpZoom">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Zoom</string>
       </property>
       <widget class="QDoubleSpinBox" name="dsbZoom">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="grpStyle">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>121</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Style</string>
       </property>
       <widget class="QRadioButton" name="rdbPointCloud">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
         </font>
        </property>
        <property name="text">
         <string>Point Cloud</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="rdbWireframe">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
         </font>
        </property>
        <property name="text">
         <string>Wireframe</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actnOpenFile"/>
    <addaction name="actnExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuPoint_Size">
     <property name="title">
      <string>Point Size</string>
     </property>
     <addaction name="actPointSize1"/>
     <addaction name="actPointSize2"/>
     <addaction name="actPointSize3"/>
     <addaction name="actPointSize4"/>
     <addaction name="actPointSize5"/>
    </widget>
    <addaction name="actnBackgroundColour"/>
    <addaction name="actnForegroundColour"/>
    <addaction name="menuPoint_Size"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actnAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actnOpenFile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/icons/open.ico</normaloff>:/images/images/icons/open.ico</iconset>
   </property>
   <property name="text">
    <string>Open File...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actnExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/icons/quit.ico</normaloff>:/images/images/icons/quit.ico</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actnBackgroundColour">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/icons/bucket.ico</normaloff>:/images/images/icons/bucket.ico</iconset>
   </property>
   <property name="text">
    <string>Background Colour</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actnForegroundColour">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/icons/paint-brush.ico</normaloff>:/images/images/icons/paint-brush.ico</iconset>
   </property>
   <property name="text">
    <string>Foreground Colour</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actnAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/icons/question.ico</normaloff>:/images/images/icons/question.ico</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actPointSize1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/icons/1.ico</normaloff>:/images/images/icons/1.ico</iconset>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="actPointSize2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/icons/2.ico</normaloff>:/images/images/icons/2.ico</iconset>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="actPointSize3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/icons/3.ico</normaloff>:/images/images/icons/3.ico</iconset>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="actPointSize4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/icons/4.ico</normaloff>:/images/images/icons/4.ico</iconset>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="actPointSize5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/icons/5.ico</normaloff>:/images/images/icons/5.ico</iconset>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ModelViewer</class>
   <extends>QWidget</extends>
   <header>modelviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
